INSTALL

```diff
@@ -1,1247 +1 @@
-  
-  Admin users:
-   
-  $ bin/turnadmin -A -N "host=localhost dbname=2 user=turn password=turn" -u gorst -p hero
-  $ bin/turnadmin -A -N "host=localhost dbname=2 user=turn password=turn" -u ninefingers -p youhavetoberealistic -r north.gov
-  
-See the file testredisdbsetup.sh for the data structure examples.
-
-XX. Performance tuning
-
-This topic is covered in the wiki page:
-
-https://github.com/coturn/coturn/wiki/TURN-Performance-and-Load-Balance
-
-XXI. TURN Server setup
-
-Read the project wiki pages: https://github.com/coturn/coturn/wiki
-
-Also, check the project from page links to the TURN/WebRTC configuration examples.
-It may give you an idea how it can be done.
-
-XXII. HTTPS Management Interface
-
-The turnserver process provides an HTTPS Web access as statistics and basic management
-interface. The turnserver listens to incoming HTTPS admin connections on the same ports
-as the main TURN/STUN listener. The Web admin pages are basic and self-explanatory.
-
-To make the HTTPS interface active, the database table admin_user must be
-populated with the admin user account(s). An admin user can be a superuser
-(if not assigned to a particular realm) or a restricted user (if assigned to
-a realm). The restricted admin users can perform only limited actions, within
-their corresponding realms.
-
-XXIII. Telnet CLI management interface
-
-You have a telnet interface (enabled by default) to access the turnserver process, 
-to view its state, to gather some statistical information, and to make some changes 
-on-the-fly.
-
-You can access that CLI interface with telnet or putty program (in telnet mode). 
-The process by default listens to port 5766 on IP address 127.0.0.1 for the telnet
-connections.
-
-WARNING: all telnet communications are going unencrypted over the network. For
-security reasons, we advise using the loopback IP addresses for CLI (127.0.0.1 
-or ::1). The CLI may have a password configured, but that password is
-transferred over the network unencrypted, too. So sticking to the local system
-CLI access, and accessing the turnserver system terminal with ssh only, would 
-be a wise decision.
-
-XXIV. ALPN support.
-
-Starting with version 4.3.2.1, the TURN server supports the ALPN STUN 
-specifications (http://tools.ietf.org/html/draft-ietf-tram-alpn-08).
-If the ALPN functionality is needed, then OpenSSL version 1.0.2 or 
-newer has to be used. See OPENSSL section for the OpenSSL upgrade hints.
-
-XXV. SCTP support
-
-Starting with version 4.4.3.1, the TURN server supports 'native' SCTP.
-On the client side, the TURN server, additionally, supports SCTP and 
-TLS-over-SCTP.
-
-The relay side is not changing - the relay communications will still be UDP
-or TCP.
-
-XXV. Prometheus support.
-
-See for source and releases for debian packages:
-https://github.com/digitalocean/prometheus-client-c
+See https://github.com/coturn/coturn/docs
```

Comment:
Because threr are the docs folder in source code tree. so that :
```
See: [docs](docs)
```

Reply:
Right, my concern is about people who install and now expect to open INSTALL file and see the documentation and it is not there - so there is a full link for them

But I agree - maybe we should have both
Eventually we may transfer all documentation online and only package man pages...

Reply:
Agree with KangLin here.
Let's make it "See docs folder" or something like that.